utils/index.js

```diff
@@ -0,0 +1,13 @@
+function hasPackage(pkgName) {
+  try {
+    require.resolve(pkgName, { paths: [process.cwd()] });
```

Comment:
We might need to use a more strict check than this. I tried in `ng-tax` which doesn't have react nor typescript. React behaved correctly, but typescript was shown as "installed":

<img width="615" alt="image" src="https://user-images.githubusercontent.com/4542735/118443422-337b5a00-b72f-11eb-9e35-cb1e69a0d428.png">